tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -614,6 +614,16 @@ public static partial class Tags
         /// </summary>
         internal const string AppSecWafVersion = "_dd.appsec.waf.version";
 
+        /// <summary>
+        /// The most relevant WAF error code during a request
+        /// </summary>
+        public const string WAFError = "_dd.appsec.waf.error";
```

Comment:
Nit picking, but to follow .NET naming conventions these should be:
```
WafError
RaspWafError
```

like other 3-letter acronyms: `Sql`, `Xml`, etc

See also `AppSecWafVersion` above and `AppSecWafInitRuleErrors` below :)

Reply:
I have fixed the tag names. Thanks for pointing this out!